caclusterissuer/README.md

```diff
@@ -0,0 +1,28 @@
+# CA Cluster Issuer Configuration
```

Comment:
We have already documented this here: 
https://gitlab.eng.vmware.com/jwolf/vsphere-with-tanzu-dev-center/-/tree/main/cert-manager#how-to-deploy-a-self-signed-ca-issuer-and-request-a-certificate

so @jswolf059 aren't we going to release the guides? I don't want 2 sources of truth for how to configure and use the service so they should at least reference each other to some extent.

Reply:
The link you provided goes into quite a bit of detail about how to generate your certificate. Do think that is necessary documentation to include? The other gotcha is that the instructions provided in that link only work on Linux. I had to write up one specifically for macOS [here](https://confluence.eng.vmware.com/display/~ecody/How+to+Generate+a+Root+CA+for+Testing+cert-manager+using+MacOS). It seems there are many cases that we would need to cover so that someone could generate their certificate. Maybe we can assume that if they are using this, they can either use the example we provide, or we can assume they know how to manage a PKI. If we make that assumption, then we don't need to provide instructions on how to generate a root CA. I created that confluence page just for my edification (and other curious parties) but there are so many options someone can provide that it would be hard to cover all the potential cases. What are your thoughts?

Reply:
The content in https://gitlab.eng.vmware.com/jwolf/vsphere-with-tanzu-dev-center/-/tree/main/cert-manager#how-to-deploy-a-self-signed-ca-issuer-and-request-a-certificate only covers how to request a certificate from a CA ClusterIssuer. The details that Cody added to the caclusterissuer/README covers what a VI Admin would need to provide in the configuration field during install (open yaml field)

Reply:
@jswolf059 We have this section https://gitlab.eng.vmware.com/jwolf/vsphere-with-tanzu-dev-center/-/tree/main/cert-manager#how-to-deploy-a-cluster-ca-issuer-and-request-a-certificate so we should link one to the other to make sure people understand.

Reply:
@AcidLeroy and we should augment the other doc to make sure all OS are covered. We do not want to assume anything, we want everyone to be able to try  knowing nothing and just following instructions.

Reply:
@fstrudel I will take an action item to update the documentation. We have to be a little careful because generating root certificates starts to get into the territory of maintaining one's own PKI. We can give developers some tools to get started, but we should caveat the process. I.e., generating a certificate yourself is just for testing purposes, but to do it properly it will take significantly more work and planning.

Reply:
The disclaimer is here already IIRC. If you provide a cluster issuer service then you need to provide full doc, otherwise don't provide cluster issuer at all...??